### DISCUSSION TITLE
Ayuda con mi práctica de pastelería

### DISCUSSION THREAD

**Discussion by eskaider** [Launch-X-Latam/MisionFrontEnd#262] [Q&A]
Compañeros cómo puedo hacer que en mis checkbox salga la imagen del pastel que estoy ofertando ya que no me sale anexo fotos para ver si alguien puede ayudarme 🥺😥 se supone que debe verse como en el wireframe pero no se me ocurre como 😞 ![image](https://user-images.githubusercontent.com/102715365/197676070-d2a150f9-4eda-445f-b4d6-04899fb3c35c.jpeg)

![image](https://user-images.githubusercontent.com/102715365/197676135-aad30b24-e705-4e1f-9b74-e8d05faa6e2b.jpeg)

![image](https://user-images.githubusercontent.com/102715365/197676143-5eda4fd0-9d99-4b3c-b1db-9ee68aff5210.jpeg)

- [ ] ![image](https://user-images.githubusercontent.com/102715365/197676150-c3952f95-e93a-49d8-a487-071988da9d65.jpeg)

**Comment by Amhedriel** [comment]
Buenas colega, te recomiendo que le des un vistazo a CSS Grid para ver mejor el modelado que al parecer quieres para tu página Web y... porqué tienes un `div` dentro de otro `div`? que quieres hacer ahi?

**Reply by eskaider** [reply]
ya vi CSS grid y la verdad me es muy complicado cambie mi codigo pero aun asi no queda como mi wireframe estoy ya demasiado frustrado porque llevo 2 dias y no puedo hacerlo ya cheque grid flexbox pero no dice nada sobre los inputs ahora el problema es que la imagen me queda a un costado cuando deberia ser arriba puse la propiedad float: left; y nada
![Css](https://user-images.githubusercontent.com/102715365/197928337-ed8d22af-9069-441c-9749-73de121aefe1.png)
![html](https://user-images.githubusercontent.com/102715365/197928345-ebc45917-63ce-46d2-8cdf-5094b437d462.png)
![pantalla](https://user-images.githubusercontent.com/102715365/197928349-b4fc393c-56c2-488d-a604-096f658c3811.png)

**Reply by eskaider** [reply]
anexo enlace de mi repositorio espero puedan ayudarme a resolver mi problema o guiarme para saberlo solucionar para futuros proyectos

https://github.com/eskaider/El-emporio-del-pastel

**Comment by Leonardo-Ismet** [comment]
Hola compañero por cada pastel  faltaría agregar esta línea para integrar la imagen 
<img class= "Pastel de fresa" src="link de referencia a la imagen" alt="Pastel de fresa" width="50px" height="50px"> 
tu cambias el tamaño o con la práctica de CSS lo vuelves resposive, ahora si lo haces con estilos podría ser algo así lo cual afectaría a las clases
 .img Pastel1 {
    display: flex
    width: 90%;
    height: 90%;
    padding: 1%;
    align-items: center;
}

Espero te sirva, saludos

**Reply by Leonardo-Ismet** [reply]
(< img class= "img" src ="Link a la foto" alt ="Pastel de fresa" width ="50px" height ="50px" >) solo retiras los espacios

**Reply by eskaider** [reply]
intente hacerlo peo no me funciona
![pantalla](https://user-images.githubusercontent.com/102715365/197929153-28a172b1-f1e2-4e19-9c01-91bcab66b963.png)
![html](https://user-images.githubusercontent.com/102715365/197929154-7ed5e454-def6-456b-82c1-e429f772c9b1.png)
![Css](https://user-images.githubusercontent.com/102715365/197929157-2c073fcf-0116-4375-9612-fbac6384eb74.png)

ahora sale asi y no puedo moverlo

**Reply by eskaider** [reply]
anexo enlace de mi repositorio espero puedan ayudarme a resolver mi problema o guiarme para saberlo solucionar para futuros proyectos

https://github.com/eskaider/El-emporio-del-pastel
